packages/openapi/src/middleware.test.ts

```diff
@@ -34,9 +35,8 @@ export function createContext<T extends Koa.Context>(
   return ctx as T
 }
 
-const PATH = '/{accountId}/incoming-payments'
-const SPEC =
-  'https://github.com/interledger/open-payments/raw/3930448672cfc678ec2bc02938566a316d83871c/open-api-spec.yaml'
+const PATH = '/incoming-payments'
+const SPEC = path.resolve(__dirname, '../../../openapi/resource-server.yaml')
```

Comment:
editor's note: a later commit will pin this to a link to a specific commit of this file, like before.